hipparchus-core/src/main/java/org/hipparchus/analysis/differentiation/FieldDerivativeStructure.java

```diff
@@ -806,9 +806,8 @@ public int getExponent() {
         return result;
     }
 
-    /** Convert radians to degrees, with error of less than 0.5 ULP
-     *  @return instance converted into degrees
-     */
+    /** {@inheritDoc} */
+    @Override
     public FieldDerivativeStructure<T> toDegrees() {
         final FieldDerivativeStructure<T> ds = factory.build();
         final double factor = 180 / FastMath.PI;
```

Comment:
Perhaps you could use here the constant RAD_TO_DEG that you defined in the interface